templates/seL4VMParameters.template.c

```diff
@@ -8,17 +8,26 @@
 #include <camkes.h>
 #include <arm_vm/gen_config.h>
 
-/*- set vm_address_config = configuration[me.name].get('vm_address_config') -*/
-/*- set vm_image_config = configuration[me.name].get('vm_image_config') -*/
-/*- set linux_address_config = configuration[me.name].get('linux_address_config') -*/
-/*- set linux_image_config = configuration[me.name].get('linux_image_config') -*/
+/*- set config = configuration[me.name] -*/
```

Comment:
I'm wondering whether these deserve a commit of their own (even in this very same PR); good stuff, but not exactly about priorities.

Reply:
moved to a separate commit